docs/en/flash.md

```diff
@@ -1,10 +1,10 @@
-Adafruit provides a really nice [firmware flashing tutorial](https://learn.adafruit.com/building-and-running-micropython-on-the-esp8266/flash-firmware). Below you'll find just the basics for the two popular tools esptool and NodeMCU Flasher.
+Adafruit provides a really nice [firmware flashing tutorial](https://learn.adafruit.com/building-and-running-micropython-on-the-esp8266/flash-firmware). Below you'll find just the basics for three popular tools: esptool.py, esptool-ck and NodeMCU Flasher.
 
 !!! attention
 
     Keep in mind that the ESP8266 needs to be [put into flash mode](#putting-device-into-flash-mode) before you can flash a new firmware!
 
-## esptool
+## esptool.py
```

Comment:
Riiiight...while the GitHub repo is called `esptool` the author(s) mostly seem to be referring to it as `esptool.py`.